Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -756,106 +739,111 @@ private final class TokenStreamCreator: SyntaxVisitor {
       preconditionFailure()
     }
 
+    let tokenToOpenWith = node.condition?.lastToken ?? node.poundKeyword
```

Comment:
This handler is for `#if` statements, rather than `if #available`. The `condition` property is optional, which appears to happen when the node is an `#else`. In that case, we need to anchor off the right token.